### PR TITLE
Add How-to change a parameter

### PR DESCRIPTION
**Pull request by ccremer**
* Adds a page that generally explains how to update a parameter and push the change.

Since I don't work with Syn everyday and the UX of commodore is not exactly self-explaining, I often forget what the workflow is to make a simple change on a cluster. This How-to guide should help anyone with the same problem.

## Checklist

- [x] Keep pull requests small so they can be easily reviewed.
- [x] Update the documentation.

### CODE REVIEW

File: docs/modules/ROOT/pages/how-tos/change-parameter.adoc [hunk 1]
```diff
@@ -0,0 +1,37 @@
+= Change a Parameter
```
Comment (simu) [inline docs/modules/ROOT/pages/how-tos/change-parameter.adoc]: According to our style guide we don't use Title Case for headings, cf. https://syn.tools/syn/references/style-guide.html#_asciidoc_style and https://docs.microsoft.com/en-us/style-guide/capitalization#sentence-style-capitalization-in-titles-and-headings

```suggestion
= Change a parameter
```

File: docs/modules/ROOT/pages/how-tos/change-parameter.adoc [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+Suppose you want to overwrite an inventory parameter for a cluster or tenant.
+This page assumes that you have already set up access to Lieutenant, so that `commodore catalog compile` generally works from any directory.
```
Comment (simu) [inline docs/modules/ROOT/pages/how-tos/change-parameter.adoc:4]: Probably want to add a note about a local installation of `commodore` somewhere in `$PATH` or an alias running dockerized Commodore, if running `commodore catalog compile` should work from any directory.

File: docs/modules/ROOT/pages/how-tos/change-parameter.adoc [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+[source,bash]
+----
+export $CLUSTER_ID = <target-cluster-id>
+dir=$(mktemp -d)
+pushd "${dir}"
+commodore catalog compile $CLUSTER_ID
+----
+. Open your editor and edit the files you need to change.
++
+TIP: Most probably the file is in `inventory/classes/<tenant-id>`
```
Comment (simu) [inline docs/modules/ROOT/pages/how-tos/change-parameter.adoc]:
```suggestion
TIP: If you want to overwrite an inventory parameter for a tenant or cluster, edit the appropriate file in `inventory/classes/<tenant-id>`.
If you want to adjust a parameter for a Kubernetes distribution or cloud provider, edit the appropriate file in `inventory/classes/global`.
+
TIP: You can compile the catalog locally to verify your changes with `commodore catalog compile $CLUSTER_ID --local`.
```

File: docs/modules/ROOT/pages/how-tos/change-parameter.adoc [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+. Navigate into the tenant directory and push a new commit
```
Comment (simu) [inline docs/modules/ROOT/pages/how-tos/change-parameter.adoc]:
```suggestion
. Navigate into the tenant or global directory, depending on which files you've edited, and commit your changes.
```

File: docs/modules/ROOT/pages/how-tos/change-parameter.adoc [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
++
+[source,bash]
+----
+pushd inventory/classes/<tenant-id>
+git status
+git add *
+git commit -m "Update cluster"
```
Comment (simu) [inline docs/modules/ROOT/pages/how-tos/change-parameter.adoc]: Please don't encourage people to use crappy commit messages

File: docs/modules/ROOT/nav.adoc [hunk 6]
```diff
@@ -36,6 +36,7 @@ include::steward:ROOT:partial$nav-tutorials.adoc[]
 
 .How-to guides
 * xref:how-tos/release-process.adoc[Release Process]
+* xref:how-tos/change-parameter.adoc[Change a Parameter]
```
Comment (simu) [inline docs/modules/ROOT/nav.adoc]:
```suggestion
* xref:how-tos/change-parameter.adoc[Change a parameter]
```

**Review by simu — CHANGES_REQUESTED** [review]

File: docs/modules/ROOT/pages/how-tos/change-parameter.adoc (hunk 5 above)
Comment (ccremer) [reply docs/modules/ROOT/pages/how-tos/change-parameter.adoc]: Removed the -m flag, that should open an editor instead.

**Review by tobru — APPROVED** [review]

File: docs/modules/ROOT/pages/how-tos/change-parameter.adoc (hunk 3 above)
Comment (simu) [reply docs/modules/ROOT/pages/how-tos/change-parameter.adoc]: Any reason why you didn't include the 2nd tip (running `commodore catalog compile $CLUSTER_ID --local`) in the current version?

Comment (ccremer) [reply docs/modules/ROOT/pages/how-tos/change-parameter.adoc]: no, I overlooked that :see_no_evil:

Comment (ccremer) [reply docs/modules/ROOT/pages/how-tos/change-parameter.adoc]: better?

**Review by simu — APPROVED** [review]

**Pull request merged**